lib/device-connections-factory.js

```diff
@@ -0,0 +1,178 @@
+import _ from 'lodash';
+import net from 'net';
+import B from 'bluebird';
+import { logger } from 'appium-support';
+import { utilities } from 'appium-ios-device';
+import { checkPortStatus } from 'portscanner';
+
+const log = logger.getLogger('iProxy');
```

Comment:
Since there may be multiple iProxy instances running, it might be nice to have a per-instance logger with more pertinent information to distinguish what the logging is pertaining to.

Reply:
yep, this makes sense. I'll change the logging logic